### PR TITLE
pt-br translation for website-sections files

### PR DESCRIPTION
**Pull request by netrik182**

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @netrik182. Will look for help on the review. 

@rmbb could you please review this PR? 

Not sure if @erciccione knows someone else.

**Comment by erciccione** [comment]
Maybe @vp11 could take a look at this?

File: monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Sobre Monero Outreach
+
+Um grupo independente
+
+Nós somos um time de criadores, escritores, técnicos, tradutores e visionários. Monero Outreach começou em Junho de 2018 e continua ganhando novos e talentosos contribuidores. Nós somos um grupo de trabalho da comunidade descentralizada do Monero e, desta forma, não reivindicamos nenhum nível de autoridade sobre o Monero - nós estamos aqui simplesmente para seguir o mandato que estipulamos para nós mesmos, porque nós amamos o Monero.
+
+## Mandato Monero Outreach
+
+1. Impulsionar uma maior adoção e conhecimento através de uma variedade de atividades relacionadas à divulgação - particularmente relações públicas, educação e marketing.
+2. Respeitar o trabalho feito pelos Desenvolvedores Core e pelos grupos de trabalho da comunidade, os princípios fundamentais do Monero e o espírito Monero, em todos os sentidos, para que nós não prejudiquemos a integridade do que do que torna o Monero excelente.
```
Comment (Mathkamy) [inline monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md]: "do que" is doubled. My correction would be this:

2. Respeitar o trabalho feito pelos Desenvolvedores Core e pelos grupos de trabalho da comunidade, os princípios fundamentais do Monero e o espírito Monero, em todos os sentidos, para que nós não prejudiquemos a integridade do que torna o Monero excelente.

File: monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+3. Fazer do mundo um lugar melhor através do Monero. Nós somos motivados pela pergunta: "Como podemos preparar nossas crianças para viver nesse mundo, se nós não preparamos o mundo para nossas crianças?"
+
+
+## Nossa trajetória
+
+### FUNDAÇÃO
+
+- Criar sistemas de comunicação.
+- Estabelecer flux de trabalho.
```
Comment (Mathkamy) [inline monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md]: I do not see why not translate "flux".

I think "Estabelecer fluxos de trabalho." would be more appropriate.

File: monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- Desenvolver o website.
+- Entrevistar os usuários atuais e os potenciais para ganhar conhecimento relevante em estratégias futuras.
+- Atrair voluntários e contribuidores talentosos.
+- Familiarizar-se com o processo e as pessoas do projeto Monero.
+
+### ESTRUTURA
+
+- Desenvolver ferramentas para ajudar na divulgação; considerando usuários novos e atuais, comerciantes, investidores e jornalistas.
+- Fornecer exemplos de como nossos esforços agregam valor ao projeto e expandem a visão.
+- Arrecadar fundos, proteger patrocinadores e gerar receita internamente para apoiar profissionais adicionais e estabilidade de longo prazo.
```
Comment (Mathkamy) [inline monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md]: This phrase has a problem of verbal regency.

My sugestion: Arrecadar fundos, proteger patrocinadores, gerar receita internamente para apoiar profissionais adicionais e obter estabilidade a longo prazo.

File: monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- Pesquisar os indivíduos, grupos e publicações que queremos alcançar.
+- Fortalecer as relações entre indivíduos e grupos dentro do projeto Monero, encontrando maneiras de construir a identidade da equipe, apesar de sua natureza descentralizada.
+
+### OPERACIONAL
+
+- Desenvolver histórias que ressoem com um público amplo e estimulem a adoção.
+- Dispersar conteúdo original e comunicados de impressa para os principais veículos de comunicação.
```
Comment (Mathkamy) [inline monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md]: "impressa" should be replaced by "imprensa". It's the most usual term to refer to the media in portuguese.

File: monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md (hunk 2 above)
Comment (netrik182) [reply monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md]: It was a typo, thanks for pointing it out!

**Comment by netrik182** [comment]
Thanks @Mathkamy for the review. File was updated. @lh1008 and @erciccione thank you both too!

**Pull request merged**